apim/templates/api-configmap.yaml

```diff
@@ -48,14 +48,15 @@ data:
     services:
       core:
         http:
-          enabled: true
-          port: 18083
-          host: localhost
+          enabled: {{ .Values.api.http.services.core.http.enabled }}
```

Comment:
You may probably have to manage the default value to `true` for those who are not using the default `values.yml` and providing their own.

Reply:
(I mean, for backward compatibility)

Reply:
I removed the possibility to disabled the technical api, as this is not really needed, and testing a value to be nil/false/true in helm isn't an easy task (cf multiple issues as this one https://github.com/helm/helm/issues/3308)